content/03-reference/01-tools-and-interfaces/02-prisma-client/050-filtering.mdx

```diff
@@ -522,6 +524,37 @@ const result = await prisma.user.findMany({
 })
 ```
 
+## Case-insensitive filtering
+
+Case-insensitive filtering is preview feature and [can only be used with PostgreSQL](case-sensitivity#options-for-case-insensitive-filtering). Add the highlighted feature flag to enable case-insensitive filtering capabilities:
+
+```prisma
+generator client {
+  provider = "prisma-client-js"
+|   previewFeatures = ["insensitiveFilters"]
+}
+```
+
+Please help us make this feature production-ready by trying it out, [sharing your feedback](https://github.com/prisma/prisma/issues/3340) and [reporting bugs](https://www.prisma.io/docs/more/creating-bug-reports/)! The more and better feedback we receive, the earlier the feature can be released for production usage.
```

Comment:
In the release notes we tried to emphasize on you can post feedback if it works, maybe we could do the same?
 
>Please share your feedback on how this feature works for you. We are interested in both positive and negative feedback, so we know if this feature is already ready for production! (If encounter any problems, please open a new issue here).